features/bootstrap/FeatureContext.php

```diff
@@ -145,15 +124,14 @@ private static function get_process_env_variables() {
 	 * Ideally, we'd cache at the HTTP layer for more reliable tests.
 	 */
 	private static function cache_wp_files() {
-		$wp_version_suffix = ( $wp_version = getenv( 'WP_VERSION' ) ) ? "-$wp_version" : '';
-		self::$cache_dir = sys_get_temp_dir() . '/wp-cli-test-core-download-cache' . $wp_version_suffix;
+		self::$cache_dir = sys_get_temp_dir() . '/wp-cli-test-core-download-cache';
```

Comment:
I think you're removing stuff that shouldn't be removed...

Reply:
Hmm, I just run `wp scaffold package-tests scaffold-command/ --force`.
Did I make any mistake?

```
$ wp --info
PHP binary:	/usr/local/Cellar/php71/7.1.10_21/bin/php
PHP version:	7.1.10
php.ini used:	/usr/local/etc/php/7.1/php.ini
WP-CLI root dir:	phar://wp-cli.phar
WP-CLI vendor dir:	phar://wp-cli.phar/vendor
WP_CLI phar path:	/Users/miyauchi/wp-cli
WP-CLI packages dir:	/Users/miyauchi/.wp-cli/packages/
WP-CLI global config:	/Users/miyauchi/.wp-cli/config.yml
WP-CLI project config:	
WP-CLI version:	1.3.0
```

```
$ wp package list 
+--------------------+-------------------------+------------+-----------+----------------+
| name               | authors                 | version    | update    | update_version |
+--------------------+-------------------------+------------+-----------+----------------+
| vccw/scaffold-vccw |                         | 1.3.0      | none      |                |
| wp-cli/scaffold-pa | Daniel Bachhuber        | dev-master | available | dev-master     |
| ckage-command      |                         |            |           |                |
+--------------------+-------------------------+------------+-----------+----------------+
```

Also, if I update WP-CLI to nightly, I have https://github.com/wp-cli/scaffold-package-command/issues/149.

Reply:
If you run `wp scaffold package-tests` from the latest stable, then it will use out of date tests.

I've reverted your commit and run `wp scaffold package-tests` with the nightly for you (so you can update your test).